tests/acceptance/pageObjects/accountPage.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+  url: function () {
+    return this.api.launchUrl + '/index.html#/account/'
+  },
+  commands: {
+    /**
+     * like build-in navigate() but also waits for the account display element
+     * @returns {Promise}
+     */
+    navigateAndWaitTillLoaded: function () {
+      return this.navigate(this.url())
+        .waitForElementVisible('@accountDisplay')
+    },
+
+    /**
+     * Gets the account information details from the account information element on the account page
+     *
+     * @returns {object}
+     */
+    getAccountInformation: async function () {
+      const accountInformation = []
+      let accountInfoElementIds = []
+      await this.waitForElementVisible('@accountInformationElements')
+      await this.api.elements('@accountInformationElements', function (result) {
+        accountInfoElementIds = result.value
+      })
```

Comment:
await this.waitForElementVisible('@accountInformationElements')
                .api.elements('@accountInformationElements', function (result) {
                           accountInfoElementIds = result.value
                 })

Reply:
done